lib/model/folder_sendrecv.go

```diff
@@ -1273,9 +1244,9 @@ func (f *sendReceiveFolder) copierRoutine(in <-chan copyBlocksState, pullChan ch
 			buf = protocol.BufferPool.Upgrade(buf, int(block.Size))
 
 			var found bool
-			if block.HashIsSHA256() {
+			if f.Type != config.FolderTypeReceiveEncrypted {
```

Comment:
Maybe for clarity hide this behind a `func (t FolderType) HasPlaintextData() bool` or so?

Reply:
I opted to put it onto the folder config, not type. Usage in the model is a bit more succinct and in case something changes in how we represent "being locally encrypted" in the process that won't affect model code like this.

Build status can be ignored, probably need to merge main into encryption again - will do.